core/templates/pages/topic-editor-page/services/topic-editor-state.service.spec.ts

```diff
@@ -200,6 +256,15 @@ describe('Topic editor state service', function() {
       'TopicRightsBackendApiService',
       [fakeTopicRightsBackendApiService][0]);
   }));
+  beforeEach(
```

Comment:
We already have this in lines 244-245, please remove it.

Reply:
Done.